tests/Tests/200__monitor_and_manage/200__metrics/206__pvc_usage_90.robot

```diff
@@ -0,0 +1,58 @@
+*** Settings ***
+Force Tags       Smoke  Sanity
+Resource         ../../../Resources/ODS.robot
+Resource         ../../../Resources/Common.robot
+Resource         ../../../Resources/Page/ODH/JupyterHub/JupyterHubSpawner.robot
+Resource         ../../../Resources/Page/ODH/JupyterHub/JupyterLabLauncher.robot
+Library          DebugLibrary
+Library          SeleniumLibrary
+Library          JupyterLibrary
+
+*** Variables ***
+${python_code}=  SEPARATOR=\n
+...  import os
+...  with open('large_file_over90.tmp', 'wb') as fout:
+...  \t fout.write(os.urandom(1841976115))
+${alert_name}=  RHODS-PVC-Usage
+${alert_description}=  User notebook pvc usage above 90%
+
+*** Test Cases ***
+Set Up JupyterLab
```

Comment:
The current recommendation for tests cases is that they should be able to be run independently of previous run tests, so instead of having 3 tests (Set Up JupyterLab, Trigger PVC 90, Check Prometheus) I think it's better that you just have one with all the necessary steps. I mean one test for ODS-516 and another for ODS-517

The test title is better that is the same (or similar) to the one in Polarion

Also, you can simplify your test using as example the [Run Jupyter Notebook For 5 Minutes](https://github.com/red-hat-data-services/ods-ci/blob/7859e5d7c800ee18d85df133121d72e6e91458b0/tests/Tests/200__monitor_and_manage/200__metrics/201__billing_metrics.robot#L68) keyword, that uses the Common Suite Setup and Teardown and uses the "Iterative Image Test" keyword to easily run a notebook in a git repo


So you can do:
```
Verify alert RHODS-PVC-Usage is fired when user notebook pvc usage is above 90 Percent
  [Tags]  ODS-516  Sanity
  Open Browser  ${ODH_DASHBOARD_URL}  browser=${BROWSER.NAME}  options=${BROWSER.OPTIONS}
  Login To RHODS Dashboard  ${TEST_USER.USERNAME}  ${TEST_USER.PASSWORD}  ${TEST_USER.AUTH_TYPE}
  Wait for RHODS Dashboard to Load
  Iterative Image Test  s2i-generic-data-science-notebook  https://github.com/redhat-rhods-qe/ods-ci-notebooks-main  notebooks/200__monitor_and_manage/203__alerts/notebook-pvc-usage/fill-notebook-pvc-over-90.ipynb
  Capture Page Screenshot
  Prometheus.Alert Should Be Firing  ${RHODS_PROMETHEUS_URL}  ${RHODS_PROMETHEUS_TOKEN}    ${rule_group}  ${alert_90}


Verify alert RHODS-PVC-Usage is fired when user notebook pvc usage is 100 percent
  [Tags]  ODS-517  Sanity
  Open Browser  ${ODH_DASHBOARD_URL}  browser=${BROWSER.NAME}  options=${BROWSER.OPTIONS}
  Login To RHODS Dashboard  ${TEST_USER.USERNAME}  ${TEST_USER.PASSWORD}  ${TEST_USER.AUTH_TYPE}
  Wait for RHODS Dashboard to Load
  Iterative Image Test  s2i-generic-data-science-notebook  https://github.com/redhat-rhods-qe/ods-ci-notebooks-main  notebooks/200__monitor_and_manage/203__alerts/notebook-pvc-usage/fill-notebook-pvc-to-100.ipynb
  Capture Page Screenshot
  Prometheus.Alert Should Be Firing  ${RHODS_PROMETHEUS_URL}  ${RHODS_PROMETHEUS_TOKEN}    ${rule_group}  ${alert_100}
```

 

```